mmdet/apis/train.py

```diff
@@ -68,6 +68,51 @@ def set_random_seed(seed, deterministic=False):
         torch.backends.cudnn.benchmark = False
 
 
+def scale_lr(cfg, logger):
+    """Automatically scaling LR according to GPU number and sample per GPU.
+
+    Args:
+        cfg (config): training config.
+        logger (logging.Logger): logger.
+    """
+    enable_auto_scale_lr = cfg.get('enable_auto_scale_lr', False)
+    if enable_auto_scale_lr is False:
+        logger.info('You disabled automatically scaling LR.')
+        return
+
+    # default batch size of pre-train model
+    original_batch_size = cfg.get('default_batch_size', None)
+    assert original_batch_size is not None
+
+    # calculate the batch size
+    gpu_number = len(cfg.gpu_ids)
```

Comment:
`num_gpus` may be better to improve consistency with the code below.
https://github.com/open-mmlab/mmdetection/blob/70f6d9cfade4a2f0b198e4f64776521d181b28be/mmdet/apis/train.py#L104

Reply:
Done